### PR TITLE
Feature/slider

### PR DESCRIPTION
**Pull request by carolinepar**
Still have some future changes but probably good for the walking skeleton. Right now the date is an integer value instead of an actual date, so I can fix that in the future. Additionally, eventually I'm going to add an input box in addition to the slider so the user can go to a specific date.

### CODE REVIEW

File: src/components/Slider/Slider.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+import './style.css';
+
+class Slider extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.state= {
+      dateValue: '50'
+    }
+    this.handleChange = this.handleChange.bind(this);
```
Comment (Jaega) [inline src/components/Slider/Slider.js:11]: Just to make sure, getDate() does not need to be bound right?

File: debug.log [hunk 2]
```diff
@@ -0,0 +1 @@
+[1010/224345.556:ERROR:directory_reader_win.cc(43)] FindFirstFile: The system cannot find the path specified. (0x3)
```
Comment (Jaega) [inline debug.log:1]: I don't think this needs to be committed?

Comment (liruochen1998) [inline debug.log:1]: could we not commit this file to codebase?

**Review by liruochen1998 — COMMENTED** [review]
except this all look pretty good!

**Pull request closed without merging**